src/test/java/bio/terra/service/dataset/flight/ingest/IngestUtilsTest.java

```diff
@@ -66,4 +66,12 @@ public void testNoDoubleDash() {
     IngestUtils.validateBlobAzureBlobFileURL(
         "https://tdrconnectedsrc1.blob.core.windows.net/synapsetestdata/test/azure-simple--dataset-ingest-request.csv");
   }
+
+  @Test
+  public void testValidate() {
+    IngestUtils.validateBlobAzureBlobFileURL(
+        //
```

Comment:
Does this test need some clean-up?

Reply:
Yeah, sorry, that was a thing that I put into the wrong branch.  Will remove